tests/blackbox.py

```diff
@@ -373,3 +376,35 @@ def approval():
             return Seq(*(preps + results))
 
         return approval
+
+    def dryrun_on_sequence(
+        self,
+        inputs: List[Sequence[Union[str, int]]],
```

Comment:
```suggestion
        inputs: list[Sequence[str | int]],
```

Reply:
@tzaffi Addressed in https://github.com/algorand/pyteal/pull/335/commits/d799e3752e2a92f52102a06ae85f4f499419ae69.  I didn't directly take in the _suggestion_ because I had to make several changes in 1 commit.

If it looks good, can you resolve?

Reply:
👍